lib/faktory_worker/batch.ex

```diff
@@ -48,7 +48,7 @@ defmodule FaktoryWorker.Batch do
   ```
 
   """
-  alias FaktoryWorker.Job
+  alias FaktoryWorker.{Job, Sandbox}
```

Comment:
nit but can we make this individual aliases? It makes grepping harder for specific full module paths.

```suggestion
  alias FaktoryWorker.Job
  alias FaktoryWorker.Sandbox
```